Document editorial changes from January 2025. (#342)
ynput/ayon-documentation: website/docs/addon_hiero_artist.md

Product name is created dynamically if `<track_name>` is selected on **Product name**.
<br></br>

> ![Create menu](assets/resolve_review_toggle.png)

Reviewer: For reference, Here's how it looks like on my side.

I think it'd be better to update the image on the right side instead of adding an additional image.
Also, the image on the right doesn't show the review toggle.

![image](https://github.com/user-attachments/assets/7e760dcc-7a5b-4b3b-a480-b579c738ab2d)

Author: This is adjusted, to enforce this is an instance attribute and not a creator attribute.

In case you wish to publish reviewable video per `plate`:

> <<<<<<< Updated upstream
> 1. Enable the `review` toggle
> 2. Select either the appropriate track name or `[clip's media]` (will use the clip's own source).
> =======
> 1. Ensure the `review` toggle is enabled in the plate instance
> 2. Select either the appropriate track name or `[clip's media]` (will use the clip own source) as `Reviewable Source`
> >>>>>>> Stashed changes

Reviewer: ```suggestion
1. Ensure the `review` toggle is enabled for the plate instance
2. Select either the appropriate track name or `[clip's media]` (will use the clip own source) as `Reviewable Source`
```

This video shows a way to publish shot look as effect from Hiero to Nuke.

<iframe width="512px" height="288px" src="https://www.youtube.com/embed/HzZDdtII5io" frameborder="0" modestbranding="1" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="1"></iframe>

:::note
If you are not interested in publishing `effects`, this feature can be opt out in the Publisher UI through a toggle on each `plate` product instance.
:::

### Restrict instances collection to current clip selection

With long sequences, it is often the case that dozens (hundered?) of `plate`, `shot` and `audio` instances are created on a single timeline.

In that case, republishing only part of the sequence might be challenging, once would need to go through the product instances list and manually enable/disable the relevant ones.

To ease this corner case, the user can leverage the `ayon+settings://hiero/create/CollectShotClip/collectSelectedInstance` settings which, when enabled, restrict the product instance collection to the selected clips from the timeline.

:::note

> If current selection does not contain any clip, then all instances are collected.

Reviewer 2: ```suggestion
If the current selection does not contain any clip, then all instances are collected.
```
